exercises/concept/troll-the-trolls/.docs/hints.md

```diff
@@ -0,0 +1,28 @@
+# Hints
+
+## 1. Set up user types and permissions.
+
+- To handle each `AccountStatus` type, one could use an `if` statement, but the [`switch` statement][switch] is a great alternative.
```

Comment:
"each AccountStatus type" -> "each AccountStatus enumerator" or just "each AccountStatus"
either the more formal "one" or the more personal "you", but not both in this file.
"*an* if statement" / "*the* switch statement" -> either "a/an" or "the" for both